src/nativescript/nativescript-sql-module.ts

```diff
@@ -7,9 +7,11 @@ const NativeScriptSQLite = require('nativescript-sqlite');
 
 export class NativescriptSqlModule {
   private _databaseName: string;
+  private _encryptionKey: string;
 
-  constructor(databaseName) {
+  constructor(databaseName, encryptionKey) {
```

Comment:
Is this a breaking change, or does the module just create an unencrypted database if the `encyrptionKey` is null?

Reply:
This is not a breaking change. An unencrypted database will be created if `encryptionKey` is `null`. I have tested this situation.